tests/fixtures/.env.test

```
@@ -0,0 +1,2 @@
+APP_ENV=test
+API_BASE_PATH=/v3
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_BASE_PATH key should go before the APP_ENV key

(UnorderedKey)